scripts/measurements/darshini-test.sh

```diff
@@ -3,34 +3,33 @@
 # Purpose: measure the impact of concurrency on darshini
 # Date: 01-Aug-2017
 # Dependencies: Darshini must be installed and running
-#		Darshini logs are stored in a file named "darshini"
-# Invocation: $./darshini-test.sh as root user
+# Invocation: $./darshini-test.sh as non-root user
 # Output: results in "darshini-test.log"
 ######################
 
-truncate -s 0 darshini-test.log
+truncate -s 0 data/log/darshini-test.log
 
-echo "performance of Darshini on four processors"  >> darshini-test.log
-echo "--------------------------------------" >> darshini-test.log
 
-for pcapFile in `ls Test_pcap_files/sample_capture_test*.pcap`
+echo "--------------------------------------" >> data/log/darshini-test.log
+dir=$(pwd)
+for pcapFile in `ls $dir/data/packet/*.pcap`
 do
     #perform a trial run to get all the data into RAM
     /usr/bin/time bro -r $pcapFile > /dev/null 2>&1
 
-    truncate -s 0 darshini-logs/darshini
+    
         
     cpus=$(cat /sys/devices/system/cpu/online)
 
-    echo -e "\n\n\nExperiment"  >> darshini-test.log
-    echo -e "---------------" >> darshini-test.log
-    echo -e "file name = $pcapFile, number of cpus=$cpus" >> darshini-test.log
-    echo -e "\n\n"  >> darshini-test.log
+    echo -e "\n\n\nExperiment"  >> data/log/darshini-test.log
+    echo -e "---------------" >> data/log/darshini-test.log
+    echo -e "file name = $pcapFile, number of cpus=$cpus" >> data/log/darshini-test.log
+    echo -e "\n\n"  >> data/log/darshini-test.log
     #run one experiment in Darshini
     curl -H "Content-Type: application/json" -d '{"email": "abc", "password": "abc"}' http://localhost:8080/protocolanalyzer/signin
-    curl -X GET -H "Content-Type: application/json" http://localhost:8080/protocolanalyzer/test?pcapPath=\'$pcapFile\'
+    curl -X GET -H "Content-Type: application/json" http://localhost:8080/protocolanalyzer/test?pcapPath=$pcapFile
     sleep 10
-    cat darshini-logs/darshini >> darshini-test.log
```

Comment:
@raghavagrawal why is this line removed? The line helps collect the the analysis log from BITS Darshini webapp into darshini-test.log

Reply:
I will again put in this line. This was some misunderstanding.